src/main/scala/util/DFT.scala

```diff
@@ -0,0 +1,27 @@
+// See LICENSE for license details.
+package sifive.blocks.util
+
+import chisel3._
+
+import freechips.rocketchip.config.{Parameters}
+import freechips.rocketchip.diplomacy._
+
+trait HasDFT[T <: Data] {
```

Comment:
I'm assuming this needs to be type templated because not all devices have the same DFT ports, and in that case I think `T <: Bundle` is more specific

Reply:
I'm change trait name and type parameters

And also change `makePort` method name to `makeDFTPort`

Thank you for review this code :)